sdk/core/Azure.Core.TestFramework/README.md

```diff
@@ -402,3 +402,35 @@ To download and unpack all artifacts use the `Download-DevOpsRecordings.ps1` scr
 The `Download-DevOpsRecordings.ps1` would wait for active runs to finish before retrieving artifacts unless `-NoWait` switch is used.
 
 **NOTE:** these scripts require being signed in with Azure CLI (https://docs.microsoft.com/cli/azure/authenticate-azure-cli?view=azure-cli-latest) and access to the internal DevOps project (https://dev.azure.com/azure-sdk/internal/) 
+
+## Note on sharing fields between Generated and Customized code (such as _clientDiagnostics)
```

Comment:
This is more about private fields in general than the code gen split.  If you added your own by hand, they'd have the same problem which makes it even more confusing.  :smile:

```suggestion
## Note on private/non-virtual fields in your clients (such as _clientDiagnostics)
```